packages/@n8n/typescript-config/modern/tsconfig.cjs.json

```diff
@@ -0,0 +1,9 @@
+{
+	"compilerOptions": {
+		"composite": true,
+		"module": "umd",
```

Comment:
should this be commonjs?

Reply:
Commonjs is meant to be run server-side, while umd is meant to be universal, suitable for both commonjs server-side and frontend imports.